docs/doc/12-load-data/06-data-load-transform.md

```diff
@@ -2,81 +2,132 @@
 title: Transforming Data During Load
 ---
 
-Databend provides a powerful feature that allows you to transform data while loading it into a table using the COPY INTO command. This functionality simplifies your ETL (extract, transform, load) pipeline with basic transformations. By transforming data during a load, you can avoid the use of temporary tables, streamline your ETL process, and improve performance.
+Databend provides a powerful feature that allows you to transform data while loading it into a table using the [COPY INTO](../14-sql-commands/10-dml/dml-copy-into-table.md) command. This functionality simplifies your ETL (extract, transform, load) pipeline with basic transformations. By transforming data during a load, you can avoid the use of temporary tables, streamline your ETL process, and improve performance.
 
 The COPY INTO command supports column reordering, column omission, and casts using a SELECT statement. This means that your data files don't need to have the same number and ordering of columns as your target table. You can also convert data types during the load, which is particularly useful if you need to change the format of data in your source files to match the data type of the target table.
 
 :::note
 This feature is currently only available for the Parquet file format.
 :::
 
-## Loading Subset of Table Data
+## Tutorials
 
-The following example loads data from columns `id`, `name` of a staged Parquet file:
+This section provides three brief tutorials that offer practical guidance on how to transform data while loading it. Please note that these tutorials are independent of each other, and you don't need to complete them in order. Feel free to follow along based on your needs.
 
-**Sample Data**
+### Before You Begin
 
-```text
-id | name       | age
----|------------|----
- 1 | John Doe   |  35
- 2 | Jane Smith |  28
+Download the sample file [employees.parquet](https://datasets.databend.rs/employees.parquet) and then upload it to your user stage using the [File Upload API](../11-integrations/00-api/10-put-to-stage.md). If you query the file after upload, you will find that it contains these records:
+
+```sql
+SELECT * FROM @~/employees.parquet;
+
```

Comment:
![image](https://user-images.githubusercontent.com/172204/236500172-13e99188-3327-47e8-a6bb-44a0a43f5376.png)
This parquet file looks like have only one record.

Reply:
It works in my local deployment:
<img width="1209" alt="Screenshot 2023-05-05 at 8 29 41 PM" src="https://user-images.githubusercontent.com/106025534/236588486-a39ed268-957a-4eff-a781-1cc72693be9d.png">